packages/blade/src/components/Spinner/BaseSpinner/SpinningBox.web.tsx

```diff
@@ -14,6 +14,7 @@ const rotate = keyframes`
 
 const SpinningBox = styled(Box)(({ theme }) => {
   return css`
+    padding: 1px;
```

Comment:
Why is this needed?

Reply:
@kamleshchandnani ^

Reply:
on design we added this to make it visually aligned with text previously without this the spinners were looking weird when placed alongside textinput or menu items

Reply:
But this also means that when Spinners are used standalone for example in center of a page, it will be 1px off 🤔 

@saurav12  is that okay?

Reply:
@chaitanyadeorukhkar Yes, that is intentional. And yes while it is rendered standalone on a page, it is okay to render it with `1px` padding.